pom.xml.asc

```diff
@@ -0,0 +1,16 @@
+-----BEGIN PGP SIGNATURE-----
+
```

Comment:
Please verify if the pom.xml.asc file is generally committed to a repo. I'm not sure if this contains private key(s) and such.